obsah/__init__.py

```diff
@@ -178,16 +178,15 @@ def metadata_name():
         """
         return 'metadata.obsah.yaml'
 
-
     @staticmethod
     def data_path():
         """
         Return the data path. Houses playbooks and configs.
         """
         path = os.environ.get('OBSAH_DATA')
         if path is None:
-            # TODO: deprecated in 3.11
-            path = resources.path(__name__, 'data')
+            with resources.path(__name__, 'data') as fspath:
```

Comment:
While playing with this I noticed this is rather a pain to to use because `__name__` is `obsah`, not whatever the caller is. Effectively it means everyone needs to override this method.

Reply:
And we do :(

https://github.com/theforeman/obal/blob/c9f0a07b4cbfc230c057ba3edc39cee66732a0ea/obal/__init__.py#L43

Reply:
Yes, it's why we didn't notice this regression.